syftbox/version_matrix.json

```diff
@@ -0,0 +1,39 @@
+{
+	"0.1.0": ["0.1.0", "0.2.0"],
+	"0.1.1": ["0.1.0", "0.2.0"],
+	"0.1.2": ["0.1.0", "0.2.0"],
+	"0.1.3": ["0.1.0", "0.2.0"],
+	"0.1.4": ["0.1.0", "0.2.0"],
```

Comment:
can you make sure its clear that this is a server-client version matrix, either by adding a key or changing the filename?